apollo-compiler/src/test/graphql/com/example/measurements

```diff
@@ -1,58 +0,0 @@
-fragment_in_fragment                                              100ms                1.87s
-root_query_fragment_with_nested_fragments                        74.9ms                1.27s
```

Comment:
Would be nice to add the number of generated lines to this file so we can compare. I'll do something

Reply:
no worries I will do that, should be trivial